harness/determined/common/experimental/determined.py

```diff
@@ -307,8 +330,14 @@ def get_models(
                 include models with names matching this parameter.
             description: If this parameter is set, models will be filtered to
                 only include models with descriptions matching this parameter.
-            model_id: If this paramter is set, models will be filtered to
+            model_id: If this parameter is set, models will be filtered to
```

Comment:
good catch!

i noticed several other non-*.rst files where "parameter" or "hyperparameter" is spelled "paramter" or "hyperparamter" including the following:

- asha.py
- asha custom search method README.md
- determined.py
- plus some "go" files